tripy/examples/segment-anything-model-v2/README.md

```diff
@@ -22,6 +22,16 @@ This is an implementation of SAM2 model ([original repository](https://github.co
     python3 image_demo.py
     ```
 
+    <!-- Tripy: TEST: EXPECTED_STDOUT Start -->
+    <!--
+    ```
+    Generating image embedding took {137.81±10%} ms per run (averaged over 100 runs, with 5 warmup runs)
```

Comment:
The non-ascii character might give us some trouble. Can we use some other syntax? Maybe `{137.81,~10%}`